cdk/lib/frontend.ts

```diff
@@ -220,53 +230,6 @@ export class Frontend extends GuStack {
       snsTopicName: "reader-revenue-dev",
     });
 
-    new GuAlarm(this, "High5XXRateAlarm", {
-      app,
-      alarmName: alarmName(
-        "support-frontend instances are returning 5XX errors"
-      ),
-      alarmDescription: alarmDescription(
-        "Some or all actions on support website are failing"
-      ),
-      actionsEnabled: alarmEnabledInProd,
```

Comment:
Hey @jacobwinch we have this stage dependent value `alarmEnabledInProd` used for all of our custom alarms. I can see that introducing the default 5xx alarm in the `GuEc2App` pattern has added a very similar `alarmActionsEnabled` variable - is there a way to get access to that to avoid redefining it?

Reply:
It's not trivial to access this variable at the moment and as it happens we are about to move away from the current approach (i.e. using mappings) entirely. There's a bit more context on that [here](https://github.com/guardian/cdk/blob/main/docs/architecture-decision-records/006-stage-awareness.md).

Given all of the above, I think this duplication is fine for now!

Reply:
Sounds good & thanks for linking to that doc - it was an interesting read!